.github/workflows/ci.yml

```diff
@@ -19,11 +19,14 @@ jobs:
       # ESLint and Prettier must be in `package.json`
       - name: Install Node.js dependencies
         run: yarn
-
+      
       - name: Run linters
         uses: wearerequired/lint-action@v1
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           # Enable linters
           eslint: true
           prettier: true
+
+      - name: Build project
```

Comment:
Should this be a part of the `run-linters` job?

Reply:
Hehe:) Might have "koked" a little bit too much